src/test/resources/api_response_get_project_with_missing_fields.json

```diff
@@ -10,6 +10,9 @@
   ],
   "title": "Example Title",
   "language": "https://lexvo.org/id/iso639-3/nno",
+  "alternativeTitles": [],
   "startDate": "2018-11-01T00:00:00Z",
-  "endDate": "2021-05-01T00:00:00Z"
+  "endDate": "2021-05-01T00:00:00Z",
+  "grant": [],
```

Comment:
Did you update the @context to ensure that properties that are collections are annotated @container: @set?

Reply:
I did not know that was part of the specification. We need to discuss that

Reply:
After discussion: This will be solved in a later issue. 

Another conclusion: We will make some changes by using plural names for lists and send values with no content as null instead of not sending at all. This makes it better for clients